### ISSUE TITLE
Extra file

### ISSUE THREAD

**Issue by botiapa** [ammen99/wf-recorder#128]
wf-recorder works as expected, except that it also makes a `recording.mp4` file . How can I disable the creation of the file?

I'm using the following command: `wf-recorder -x yuv420p -c rawvideo -m v4l2 -f /dev/video50`

**Comment by ammen99** [comment]
Are you sure the `recording.mp4` file is not a leftover from a previous run of wf-recorder?

**Comment by botiapa** [comment]
Yes, I'm sure.

**Comment by ammen99** [comment]
This is very strange. `recording.mp4` is the default file name, it comes from this line:

https://github.com/ammen99/wf-recorder/blob/master/src/main.cpp#L657

However, if `-f` is present on the command line, it will be immediately changed to what you have specified:

https://github.com/ammen99/wf-recorder/blob/master/src/main.cpp#L694

Thus, I am not sure where ffmpeg gets that file name. Does it happen if you record to a file (e.g. `-f test.mkv`)?

**Comment by botiapa** [comment]
No, `wf-recorder -f test.mkv` writes only into the test.mkv file.
